tests/x11/updates_gnome.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub pre_run_hook() {
+    # Turn off screensaver
+    x11_start_program("xterm");
+    script_run("gsettings set org.gnome.desktop.session idle-delay 0");
+    send_key("ctrl-d");
+}
+
+sub run() {
+    my @updates_tags           = qw/updates_none updates_available/;
+    my @updates_installed_tags = qw/updates_none updates_installed-logout updates_installed-restart/;
+
+    for (1 .. 2) {
+        x11_start_program("gpk-update-viewer");
+
+        assert_screen \@updates_tags, 100;
+        if (match_has_tag("updates_none")) {
+            send_key "ret";
+            last;
+        }
+        elsif (match_has_tag("updates_available")) {
+            assert_and_click("updates_click-install");
+
+            # Authenticate on SLES
+            if (check_var("DISTRI", "sle") && get_var('FLAVOR', '') =~ /^Server/) {
+                assert_screen("updates_authenticate", 10);
```

Comment:
better than specifying a shorter then default timeout with good reasoning I suggest to just use the default.